src/Containers/StoreHoursList/Alert.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import { CheckCircle } from '@styled-icons/boxicons-regular/CheckCircle';
+import { Card } from '../Card';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+import { styledCondition } from '../../Utils/Mixins';
+
+interface AlertProps extends 
+    MainInterface,
+        ResponsiveInterface,
+        React.HTMLAttributes<HTMLSpanElement> {
+            text?: string,
+            icon?: React.ForwardRefExoticComponent<React.RefAttributes<SVGSVGElement>>,
+            success?: boolean,
+            error?: boolean
+};
+
+export const Alert: React.FC<AlertProps> = ({
+    text = 'Some text here',
```

Comment:
Shouldn't this be a required prop? With no default parameter?

Reply:
I was not able to pass it as a required prop without it showing me errors (without a default parameter). Here's the log: 
![image](https://user-images.githubusercontent.com/62865150/92549308-f3bab500-f226-11ea-8648-62ef8c20729d.png)
I passed it as children instead to fix the type error.